lib/srv/reexec.go

```diff
@@ -412,6 +412,18 @@ func RunForward() (io.Writer, int, error) {
 	return ioutil.Discard, teleport.RemoteCommandSuccess, nil
 }
 
+// runCheckHomeDir check's if the active user's $HOME dir exists.
+func runCheckHomeDir() (io.Writer, int, error) {
```

Comment:
In this case, I might name the return values.

Also, returning a nil writer feels like it could cause an issue, as the caller never checks if `w != nil`.  How about returning `io.Discard` instead of nil

Reply:
I'll also update `RunForward` and `RunCommand` to use named returns to keep it consistent.